CONTRIBUTING.md

```diff
@@ -22,12 +22,21 @@ If you have been assigned to fix an issue or develop a new feature, please follo
   - We use [nvm](https://github.com/nvm-sh/nvm) to manage node versions - please make sure to use the version mentioned in `.nvmrc`.
 - Run development server using `pnpm run watch`.
 - Implement your changes and tests to files in the `src/` directory and corresponding test files.
-- To run examples, follow their individual directions.
-- To run examples using your local build, be sure to have development server (`pnpm run watch`) running.
 - Document your changes in the appropriate doc page.
 - Git stage your required changes and commit (see below commit guidelines).
 - Submit PR for review.
 
+### Running examples
+- Make sure you've installed the dependencies by running `$ pnpm install` in the repo's root directory.
+- If you want to run the example against your local changes, run `pnpm run build` in the repo's root directory. Otherwise, it will be run against the latest TanStack Query release.
```

Comment:
Shouldn't this say `pnpm run watch` rather than `pnpm run build`?
I've tried with the watch command and it worked

Reply:
Yeah, of course! I'll fix it in a bit

Reply:
[87e5d1b](https://github.com/TanStack/query/pull/4818/commits/87e5d1b30e26d7296474fdf7783d4d70fec12920)